src/task_scl_eff_pot_hab.py

```diff
@@ -3,180 +3,200 @@
 from datetime import datetime, timezone
 from task_base import SCLTask
 
-import os
-import json
-service_account_key = os.environ.get("SERVICE_ACCOUNT_KEY")
-service_account_name = json.loads(service_account_key)["client_email"]
-credentials = ee.ServiceAccountCredentials(
-    service_account_name, key_data=service_account_key
-)
-ee.Initialize(credentials)
-
-# sumatra_aoi = ee.FeatureCollection("projects/SCL/v1/Panthera_tigris/sumatra_poc_aoi")
 
 class SCLPolygons(SCLTask):
     ee_rootdir = "projects/SCL/v1"
+    scale = 300
     inputs = {
         "structural_habitat": {
             "ee_type": SCLTask.IMAGECOLLECTION,
             "ee_path": "projects/SCL/v1/Panthera_tigris/geographies/Sumatra/hab/structural_habitat",
-            "maxage": 1,  # years
+            "maxage": 5,  # years
```

Comment:
This should stay `1` shouldn't it? We do have annual images for the years we need, for Sumatra...

Reply:
the structural_habitat only goes through 2018, and the hii through 2017. So to run currently run task with no -d argument, the maxage needs to be >1 for the moment. I included comments in the code to remind us to change when full input collections are available